frontend/packages/console-shared/src/components/status/PopoverStatus.tsx

```diff
@@ -2,16 +2,31 @@ import * as React from 'react';
 import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
 import StatusIconAndText from './StatusIconAndText';
 
-const PopoverStatus: React.FC<React.ComponentProps<typeof StatusIconAndText>> = ({
-  title,
-  children,
-  ...other
-}) => (
-  <Popover position={PopoverPosition.right} headerContent={title} bodyContent={children}>
-    <Button variant="link" isInline>
-      <StatusIconAndText {...other} title={title} />
-    </Button>
-  </Popover>
-);
+const PopoverStatus: React.FC<
+  React.ComponentProps<typeof StatusIconAndText> & {
+    activeIcon?: React.ReactElement;
+    hideHeader?: boolean;
+  }
+> = ({ title, hideHeader, icon, activeIcon, children, ...other }) => {
+  const [isActive, setActive] = React.useState(false);
+  return (
+    <Popover
+      position={PopoverPosition.right}
+      headerContent={!hideHeader && title}
+      bodyContent={children}
+      aria-label={title}
+      onHide={() => setActive(false)}
```

Comment:
Can we use [`React.useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) for function props, like `onHide` and `onShow` ?